doc/topics/releases/2015.5.4.rst

```diff
@@ -11,3 +11,7 @@ Changes:
 
 - When querying for VMs in ``ditigal_ocean_v2.py``, the number of VMs to include in a page was changed from 20
```

Comment:
No one seems to have picked up the typo of ``ditigal_ocean_v2.py`` vs ``digital_ocean_v2.py``